src/components/Day.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import activityPropTypeShape from '../prop-types/activity';
+import ActivitiesListItem from './ActivitiesListItem';
+
+const Day = ({ activities, weekDay }) => (
+  <div>
+    <p>{weekDay.toUpperCase()}</p>
+    {activities.map(activity => (
+      <ActivitiesListItem {...activity} key={activity.id} />
+    ))}
+  </div>
+);
+
+Day.defaultProps = {
```

Comment:
to have a prettier code, always put props in the same order. I feel, that more often (and more natural for me) is to have `defaultProps` at the bottom.